cmd/vsphere-xcopy-volume-populator/certificate-tool/go.mod

```diff
@@ -0,0 +1,53 @@
+module certificate-tool
+
+go 1.23.0
```

Comment:
@rgolangh we started distributing the 1.23 build image so this is no longer a blocker on our side, and if needed,d we can also bump the other parts of the offload plugin.
https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690

Reply:
it is not that important for that tool. Lets follow the cmd/vsphere-xcopy-volume-populator/go.mod version, which is 1.22 for now.